packages/ingest-mongodb-public/src/sources/mongodb-university/MongoDbUniversityDataApiClient.ts

```diff
@@ -28,8 +28,12 @@ export interface TiCatalogItem {
     IDs of labs associated with the catalog item.
    */
   associated_labs: string[];
-  // TODO: is this always null?
-  associated_content: null | unknown;
+  /*
+    IDs of content associated with the catalog item. 
+    This is used for Learning Paths and Courses. Other learning formats 
+    do not have associated content.
+   */
+  associated_content?: string[] | null;
```

Comment:
```suggestion
  associatedContent?: string[] | null;
```

camelCase!

Reply:
this is the API response, it comes in as snakecase

Reply:
ah ok, disregard then. appologies